ironic/drivers/modules/drac/raid.py

```diff
@@ -884,6 +972,65 @@ def _check_node_raid_jobs(self, task):
                 self._clear_raid_config_job_failure(node)
                 self._set_clean_failed(task, config_job)
 
+        if finished_job_ids and not node.driver_internal_info.get(
+                'raid_config_job_failure'):
+            if 'raid_config_substep' in node.driver_internal_info:
```

Comment:
There needs to be an else: clause for the if statement above that calls: self._resume_cleaning(task)

Reply:
This means we need to remove 'raid_config_substep' from node.driver_internal_info when it completes the last substep(change phy disk to JBOD).
Because if we add else cause in above if statement(line-977), 'raid_config_substep' should be removed, after than only it will execute self.resume_cleaning(task)

Reply:
Updated in new commit.